<?xml version="1.0"?>
<launch>

    <!-- the ui control node -->
    <node name="robot_position_control_node" pkg="main_node3" type="main_88" output="screen" />
    <!-- the moveit control node -->
    <node name="moveit_robotInfo_receive" pkg="main_node3" type="qnode_receive" output="screen" />


</launch>
